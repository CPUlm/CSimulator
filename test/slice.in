3
11100
01010
00001
