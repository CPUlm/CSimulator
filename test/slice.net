INPUT a
OUTPUT b, c, d
VAR
  a : 5, b : 2, c : 5, d : 3
IN
b = SLICE 1 2 a
c = SLICE 0 4 a
d = SLICE 2 4 a

