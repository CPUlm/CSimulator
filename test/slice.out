Step 1: 
=> b = 11
=> c = 11100
=> d = 100
Step 2: 
=> b = 10
=> c = 01010
=> d = 010
Step 3: 
=> b = 00
=> c = 00001
=> d = 001
